### PR TITLE
[4.0] joomla-tabs complete todo

### PR DESCRIPTION
**Pull request by brianteeman**
Completes the @todo task as the tabs custom element has been merged. As far as I can tell the reference to modals is a copy paste error

### CODE REVIEW

**Comment by richard67** [comment]
Which reference to modals?

**Comment by brianteeman** [comment]
https://github.com/joomla/joomla-cms/pull/27917/files#diff-7a15b7cbf06195cd172a48e1ad46d9bfL35

**Comment by richard67** [comment]
I see, thanks.

**Comment by richard67** [comment]
@brianteeman How to test? Code review? Or real test?

**Comment by brianteeman** [comment]
code review and make sure tabs work as they did before

**Comment by richard67** [comment]
Thanks. Code review looks good, tabs I can test tonight after work.

File: administrator/components/com_config/tmpl/component/default.php [hunk 1]
```diff
@@ -32,9 +32,6 @@
 	HTMLHelper::_('bootstrap.framework');
```
Comment (wilsonge) [inline administrator/components/com_config/tmpl/component/default.php:32]: Do we need this anymore if we're doing the bootstrap tabs cleanup? (possible the answer is yes - but just making sure)

Comment (brianteeman) [reply administrator/components/com_config/tmpl/component/default.php:32]: Don't know but isnt that beyond the scope of this PR ?

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on d222fd88cf28e7f3f7a6d11ce182dd468b17fa88<br /><br />Code review good. Tabs still work in Global Configuration.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27917">issues.joomla.org/tracker/joomla-cms/27917</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on d222fd88cf28e7f3f7a6d11ce182dd468b17fa88<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27917">issues.joomla.org/tracker/joomla-cms/27917</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27917">issues.joomla.org/tracker/joomla-cms/27917</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thx

**Pull request merged**